src/Components/Inputs/Documentation/Inputs.stories.tsx

```diff
@@ -152,6 +152,49 @@ inputsBaseStories.add(
     },
   }
 )
+inputsBaseStories.add(
+  'Input (Clearable Text)',
+  () => {
+    const [value, setValue] = useState<string>('hello world')
+
+    return (
+      <div className="story--example">
+        <Input
+          min={number('min', 0)}
```

Comment:
Did you mean `minLength` & `maxLength`? `min` & `max` are for `type="number"`.

Reply:
the 'min' and such I just copied from another storybook story.
(for the default, plain text input).

in terms of the button disappearing, @daviddkkim  specifically requested that the 'x' button only show when there is content.  so after pressing it, there is no content so it disappears.  easy to make it always appear tho.